src/group-listing.js

```diff
@@ -0,0 +1,118 @@
+const vocab = require('solid-namespace')
+
+/**
+ * Ad Hoc ACL Group Listing
+ * @see https://github.com/solid/web-access-control-spec#groups-of-agents
+ * @class GroupListing
+ */
+class GroupListing {
+  /**
+   * @constructor
+   * @param [options={}] {Object} Options hashmap
+   * @param [options.uri] {string|NamedNode} Group URI as appears in ACL file
+   *   (e.g. `https://example.com/groups#management`)
+   * @param [options.uid] {string} Value of `vcard:hasUID` object
+   * @param [options.members={}] {Object} Hashmap of group members, by webId
+   * @param [options.listing] {string|NamedNode} Group listing document URI
+   * @param [options.rdf] {RDF} RDF library
+   * @param [options.graph] {Graph} Parsed graph of the group listing document
+   */
+  constructor (options = {}) {
+    this.uri = options.uri
+    this.uid = options.uid
+    this.members = options.members || {}
```

Comment:
Set would not be appropriate here -- the `members` object is used as a dictionary, for key-based lookups. (Using Set would require using `.find()` to achieve equivalent functionality.)